blogposts/announcing-sourcegraph-3.3.md

```diff
@@ -1,44 +1,217 @@
+
+The [GitLab](https://github.com/sourcegraph/sourcegraph/blob/master/schema/gitlab.schema.json#L38-L91) and [Bitbucket Server](https://github.com/sourcegraph/sourcegraph/blob/master/schema/bitbucket_server.schema.json#L68-L113) fields for repository selection are similar to [GitHub](https://github.com/sourcegraph/sourcegraph/blob/master/schema/github.schema.json#L38-L75), but differ based on the search and selection options in the code host's API.
+
+This has been implemented for [GitHub](https://docs.sourcegraph.com/admin/external_service/github#selecting-repositories-for-code-search), [GitLab](https://docs.sourcegraph.com/admin/external_service/gitlab#repository-syncing) and [Bitbucket Server](https://docs.sourcegraph.com/admin/external_service/bitbucket_server#repository-syncing) with additional code hosts being migrated to this model in 3.4.
+
+<div id="sentry"></div>
+
+## New Sentry integration with the Sentry Sourcegraph extension
+
+TODO: Replace screencast with Sourcegraph intro screen
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/330644055?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/330644055" target="_blank">View on Vimeo</a></p>
+</p>
+
+[Sentry](https://sentry.io/) is an [open-source](https://github.com/getsentry/sentry) error tracking tool which captures and sends error and exception notifications instantly.. When browsing code on Sourcegraph or your code host, wouldn’t it be great if you could jump straight to the Sentry records page whenever you see error handling code, without having to manually open Sentry and find it?
+
+The new [Sentry Sourcegraph extension](https://sourcegraph.com/extensions/sourcegraph/sourcegraph-sentry) does just that and runs anywhere that Sourcegraph does, such as code views on GitHub/GitLab with the [Sourcegraph browser extension\(https://docs.sourcegraph.com/integration/browser_extension).
+
+The initial version of the extension provides support for the most popular languages:
+
+- Go
+- Python
+- Java
+- JavaScript
+- TypeScript
+
+Additional languages will be added as part of each new release and if you want a particular language added, create an issue on the `sourcegraph-sentry GitHub repository`.
+
+<div id="bitbucket-fixes"></div>
+
+## Bitbucket Server integration and browser extension fixes
+
+The refactoring and fixing work for integrations which started in 3.2 continues in this release, with many Bitbucket Server integration and browser extension fixes:
+
+- [**Bitbucket server fixes**](https://github.com/sourcegraph/sourcegraph/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+label%3Abitbucket+milestone%3A3.3+)<br />
```

Comment:
This bullet has a br after the title, but the next one doesn't. Should be consistent one way or the other.

Reply:
Thanks, that's been fixed